addons/van_dml/example/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="de">
+
+<head>
+  <meta charset="utf-8">
+  <title>title</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="van-0.11.10.nomodule-min.js"></script>
+  <script src="van_dml.js"></script>
+  <style>
+  </style>
+</head>
+
+<body>
+  <script>  "use strict";
+
+    const { h1, h2, div, p, button } = van.tags;
+    const { begin, end, base, sp, css } = van
+
+    const rbutton = (...args) => {
+      let b = button(...args)
+      b.style.borderRadius = "50vh       "
+      return b
+    }
+
+    begin(document.body)
+    css(`body {
+      background-image: linear-gradient(0deg, #6ac 0%, #eff 100%);
+      background-attachment: fixed;
+      background-size: 100% 100vh;
+      padding: 20px;
+      text-align: center;
+      font-family: helvetica;
+    }`)
```

Comment:
There is currently just one example showing the usage of the new functions. Though it is not necessary, it may help as a demonstations.

If people face any problems using the css()-command, a working example might help, so I would like to leave it as it is. Without the padding and center, the result looks much worse.

Reply:
I mean, the `css` function calls can be replaced with the `style` tag function as I illustrated above: https://github.com/vanjs-org/van/pull/45#discussion_r1222197277

Reply:
What's the difference between using `css` function, and using `style` tag function shown here [#45 (comment)](https://github.com/vanjs-org/van/pull/45#discussion_r1222197277)?

If you really want to add `css` function, you can proceed that way, it's just personally I don't think we need it cuz the. same thing can be achieved with `style` tag function as well.